.gitignore

```diff
@@ -12,3 +12,5 @@ dist
 .browser-refresh
 integration/template.marko
 integration/browser.json
+/.vscode
```

Comment:
We can let these unrelated changes slip through this time, but team, I'd like us to get into the habit of creating separate PRs for anything unrelated to the main issue at hand. This may seem picky, but imagine if we have to rollback this feature commit (for whatever reason) after it is squashed, we probably wouldn't want to roll back these useful `.gitignore` updates too, but that's what would happen (and then we'd have to re-commit them again). The upgrade of skin dependency is another example of this. I know I'm being exceptionally picky in this case. Sorry ;-)

Reply:
Let me just do that

Reply:
No need. Seriously. I was just using this as an example :-)

Reply:
okay :)

Reply:
I'm 100% in agreement with this. I know I just try to get improvements in, but small PRs for minor improvements are actually good, too.

Although, I do know there are times when I don't want to forget! So, I guess we create an issue to track it, @ianmcburnie?